# LiDAR cone detector configuration file

lidar_cone_detector:
  # Topic to receive PointCloud2 LiDAR data from
  lidar_topic: "/lslidar_point_cloud"
  # Topic to publish debug point cloud to, if empty string debug topic publishing is disabled
  lidar_debug_pub: "/lidar_cone/debug"
  # Whether to display the Pangolin debug UI or not
  debug_ui: true